libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -101,6 +101,7 @@
         OPENED_MEDIA_LIBRARY,
         OPENED_BLOG_SETTINGS,
         OPENED_ACCOUNT_SETTINGS, // Future
```

Comment:
OK. I removed ALL `//future` comments in [this commit](https://github.com/wordpress-mobile/WordPress-Android/commit/1b118a748caa125abbce1c1dea632069245e0c09). Thanks!